rules/sources/personal_identification.yaml

```diff
@@ -15,7 +15,7 @@ sources:
     isSensitive: False
     sensitivity: low
     patterns:
-      - "(?i).*((?:first|given)[^\\s/(;)#|,=!>]{0,5}|full)name"
+      - "(?i).*((?:first|given|fore)[^\\s/(;)#|,=!>]{0,5}|full)name"
```

Comment:
`"(?i).*((?:first|given)[^\\s/(;)#|,=!>]{0,5}|fore|full)name"`

Reply:
done. thank you @HiteshMah-Jan